llama_index/agent/openai_agent.py

```diff
@@ -148,9 +151,11 @@ def stream_chat(
         sources = []
 
         # TODO: Support forced function call
-        chat_stream_response = StreamingAgentChatResponse(
-            chat_stream=self._llm.stream_chat(all_messages, functions=functions)
-        )
+        if functions:
+            chat_stream = self._llm.stream_chat(all_messages, functions=functions)
```

Comment:
Maybe `stream_chat(all_messages, functions=functions or None)` ?

Reply:
I don't think we are allowed to pass None in.

Reply:
nope doesn't work with None.